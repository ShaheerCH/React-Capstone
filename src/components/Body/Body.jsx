import React from 'react';
import Countries from '../Countries/Countries';
import Search from '../Filter/Filter';



const Body = () => {
  return (
    <div>
      <Search />
      <Countries />
    </div>
  )
}

export default Body