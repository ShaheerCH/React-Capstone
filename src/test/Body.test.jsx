import React from 'react';
import { Provider } from 'react-redux';
import store from '../features/configureStore';
import { MemoryRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { expect, it } from 'vitest';
import Body from '../components/Body/Body';

it('Renders the App Component', () => {
  const BodyRenderer = renderer.create(
    <Provider store={store}>
      <Body />
    </Provider>
  ).toJSON();
  expect(BodyRenderer).toMatchSnapshot()
})