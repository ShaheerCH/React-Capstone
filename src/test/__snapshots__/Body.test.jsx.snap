// Vitest Snapshot v1

exports[`Renders the App Component 1`] = `
<div>
  <div
    className="Search"
  >
    <form
      className="form_input"
      onSubmit={[Function]}
    >
      <input
        id="search"
        name="search"
        onChange={[Function]}
        placeholder="Search"
        type="search"
        value=""
      />
    </form>
  </div>
  <section
    className="grid"
  />
</div>
`;
